password-auth is the Fedora's common pam configure file, use oe common pam
configure files instead.

Upstream-Status: Pending

Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
Index: cronie-1.5.1/pam/crond
===================================================================
--- cronie-1.5.1.orig/pam/crond
+++ cronie-1.5.1/pam/crond
@@ -4,8 +4,8 @@
 #
 # Although no PAM authentication is called, auth modules
 # are used for credential setting
-auth       include    password-auth
+auth       include    common-auth
 account    required   pam_access.so
-account    include    password-auth
+account    include    common-account
 session    required   pam_loginuid.so
-session    include    password-auth
+session    include    common-session-noninteractive
